client/src/services/ApiService.js

```diff
@@ -105,6 +105,9 @@ export default {
   getAllNews() {
     return jwtInterceptor.get(baseURL + "news/");
   },
+  getPublicNews() {
```

Comment:
Vorschlag:
/home liefert immer nur die aktuellen/aktiven News zurück, also solche deren Ablaufdatum nicht überschritten wurde

/news/ liefert alle News zurück. 
/news/public gibt es nicht mehr

Reply:
cool

Reply:
Ah mist...
Nachrichten die für die Zukunft geschrieben wurden, sollten ja nicht allen Usern angezeigt werden.

Also dann behalten wir /public/ und das liefert alle Nachrichten, außer denjenigen welche den Startzeitpunkt in der Zukunft haben zurück.

Der kann dann auf ListNews ausgelesen werden.